tools/trap/HowToUseTrap.md

```diff
@@ -3,11 +3,15 @@ Here we explain how to enable build support for DUMP upload,
 and how to actually upload and parse it in the event of a crash during runtime.
 
 ## Contents
+> [Prerequisites](#prerequisites)  
 > [Build Steps](#how-to-enable-memory-dumps)  
 > [Parsing Steps](#how-to-parse-RAMDUMP)  
 > [Upload Steps](#how-to-upload-RAMDUMP-or-UserfsDUMP)  
 > [Porting Guide](#how-to-port-memory-dump-functionality)  
 
+## Prerequisites
+Install Python 3.0 or above.
```

Comment:
In TizenRT docker, python3 is already installed. 
I think it is good to mention this :)